virt/virt-4-9-release-notes.adoc

```diff
@@ -114,6 +114,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 //BZ-1963254 UEFI without Secure Boot
 
+//BZ-2007397
+* Force deleting a `virt-launcher` pod when using the hot-plug or hot-unplug feature might result in the removal of the virtual machine’s disk contents from the persistent volume claim (PVC). (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007397[*BZ#2007397*])
```

Comment:
I would refer to the feature as "disk hotplug" instead of "hot-plug or hot-unplug".  It is more correct to say "from the persistent volume" rather than the "persistent volume claim".  Otherwise, looks good.

Reply:
Re: your re-wording. We have a TP release note that refers to it with similar language (see in the Tech Preview section), and the topic uses this language as well (see: https://deploy-preview-37049--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.html#virt-hot-plugging-virtual-disks). Should these both be updated? The TP note could make it for this release but the topic rewrite would need to be an item for the next release.

Reply:
I do not think that topic needs to be changed because it mentions "disk" whenever hot-plug or hot-unplug is mentioned.  In virtualization generally, you can hot-plug other resources (such as cpus, memory, and host devices) so the note should make sure it is clear that we are talking about disks here.  Since this note talks about the *data* associated with the hot-plugged PVC, Persistent Volume is more correct as I say.  The tech preview section does not discuss data and is therefore correct in referring only to PVCs and DataVolumes.  I hope this all makes sense.